pkg/trust/flags.go

```diff
@@ -53,11 +53,11 @@ func BuildConfigFromFlags(flags *pflag.FlagSet) (*StrategyConfig, error) {
 			if err != nil {
 				return nil, err
 			}
-			cert, err := util.ParsePEMEncodedCert(data)
+			certchain, err := util.ParsePEMEncodedCertChain(data)
```

Comment:
This appears incorrect - each file should only contain a single cert, or if needed we can parse the cert chain and extract the root certificate. Either way we only want to add root ca certs to this list.

Reply:
I'm not sure that makes sense.  We need to handle the case where a user is passing in a list of roots (like system roots, or a similar file)